controllers/comments.controller.js

```diff
@@ -0,0 +1,17 @@
+const { selectArticle } = require('../models/articles.modles');
+const { selectCommentsForArticle } = require('../models/comments.models');
+
+exports.getCommentsForArticle = (req, res, next) => {
+  const { article_id } = req.params;
+  const promises = [
+    selectCommentsForArticle(article_id),
+    selectArticle(article_id),
```

Comment:
Great use of your previous selectArticles function here. Also a really nice use of Promise.all()